cmd/traffic_manager/metrics.cc

```diff
@@ -340,10 +340,10 @@ metrics_binding_destroy(BindingInstance &binding)
 bool
 metrics_binding_configure(BindingInstance &binding)
 {
-  ats_scoped_str sysconfdir(RecConfigReadConfigDir());
-  ats_scoped_str config(Layout::get()->relative_to(sysconfdir, "metrics.config"));
+  std::string sysconfdir(RecConfigReadConfigDir());
```

Comment:
Why declare `sysconfdir`? Previous it would have been to have `ats_scoped_ptr` clean up correctly but `std::string` should be more robust in that regard.